src/test/java/programminglife/controller/BookmarkControllerTest.java

```diff
@@ -59,13 +73,14 @@ public void readAllGraphTest() {
     public void writeTest() {
         BookmarkController.storeBookmark(testPath, new Bookmark("dummypath", "writeTestGraph",2, 4, "writeTest", "writing"));
         BookmarkController.storeBookmark(testPath, new Bookmark("dummypath", "writeTestGraph", 2, 4, "writeTeest", "writing"));
-        assertNotNull(BookmarkController.loadAllGraphBookmarks(testPath, "writeTestGraph"));
+        assertNotNull(BookmarkController.loadAllBookmarks(testPath));
```

Comment:
This test will pass if the previous two storebookmarks don't do anything. Please use `BookarkController.loadAllBookmarks(testPath).get("dummypath")`. And if you want to be complete switch the path and graph name in the storeBookmark methods and use `.get("writeTestGraph")` instead.